### PR TITLE
MYMATH-1: Implement prime check

### PR DESCRIPTION
**Pull request by oszedinacubix**

### CODE REVIEW

File: is_prime.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def is_prime(n):
+    if n <= 1:
+        return False
+    for i in range(2, n):
```
Comment (oszedi) [inline is_prime.py:4]: run the cycle only till square root of n

File: is_prime.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        if n % i == 0:
+            return False
+    return True
+
+for i in range(1, 20):
```
Comment (oszedi) [inline is_prime.py:9]: add if __name__ == '__main__'

**Review by oszedi — CHANGES_REQUESTED** [review]
Only a few findings, otherwise nice code.

File: is_prime.py (hunk 1 above)
Comment (oszedinacubix) [reply is_prime.py]: cycle changed

File: is_prime.py (hunk 2 above)
Comment (oszedinacubix) [reply is_prime.py]: added

File: is_prime.py (hunk 1 above)
Comment (oszedi) [reply is_prime.py]: please use square root +1, otherwise it does not work for square numbers

**Review by oszedi — APPROVED** [review]
Great, thank you!

File: is_prime.py (hunk 1 above)
Comment (oszedinacubix) [reply is_prime.py]: done

**Pull request merged**